### ISSUE TITLE
Wont work with newest laravel/debugbar

### ISSUE THREAD

**Issue by schneider-simon** [scottbedard/oc-debugbar-plugin#16]
The current version of laravel/debugbar uses the illuminate/support package for Laravel 5.4.* or 5.5.*.

There are various problems considering the miss match between interfaces and the implementations (for example the FileStore class that implements the Store interface).

Installing this plugin with the latest October CMS, which facilitates Laravel 5.1, will result in the following error message:

```
2017/05/04 12:26:47 [error] 11#11: *53 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Class Illuminate\Cache\FileStore contains 2 abstract methods and must therefore be$
PHP message: PHP Fatal error:  Uncaught Error: Cannot instantiate abstract class Illuminate\Cache\FileStore in /vendor/laravel/framework/src/Illuminate/Cache/CacheManager.ph$
Stack trace:
#0 /vendor/laravel/framework/src/Illuminate/Cache/CacheManager.php(102): Illuminate\Cache\CacheManager->createFileDriver(Array)
#1 /vendor/laravel/framework/src/Illuminate/Cache/CacheManager.php(77): Illuminate\Cache\CacheManager->resolve('file')
#2 /vendor/laravel/framework/src/Illuminate/Cache/CacheManager.php(55): Illuminate\Cache\CacheManager->get('file')
...

```

The easiest fix for this problem is to replace the following line in the composer.json:
`      
  "barryvdh/laravel-debugbar": "~2.0"
`

with

`      
  "barryvdh/laravel-debugbar": "2.0.6"
`

and rerun `composer install` inside the plugins root directory.

**Comment by schempil** [comment]
+1 👍

**Comment by yapsr** [comment]
+1 Great!

**Comment by scottbedard** [comment]
Great, thanks for the help @triggerdesign !

Closed by b4499ede4cae6a66263c7b52ab0b41730b3e9b7b
